src/stan_math_backend/Transform_Mir.ml

```diff
@@ -718,6 +719,7 @@ let trans_prog (p : Program.Typed.t) =
         p.output_vars in
     { p with
       log_prob= log_prob @ maybe_add_opencl_events_clear
+    ; reverse_mode_log_prob= log_prob @ maybe_add_opencl_events_clear
```

Comment:
I think it's worth adding a comment that right here is the first time reverse_mode_log_prob is initialized, hence assigning it to log prob is not a typo/error